setup.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-# pylint: disable=C0111,W6005,W6100
+# pylint: disable=C0111,W6005,W6100,useless-suppression
```

Comment:
What was this necessary for?

Reply:
IIRC, pylint on python 3 was giving a different warning than under python 2. To make them both happy, I had to suppress a suppression.